app/components/primer/breadcrumb/item_component.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Primer
+  module Breadcrumb
+    # This component is part of `Primer::BreadcrumbComponent` and should not be
+    # used as a standalone component.
```

Comment:
Where did we land on definging this class within the parent to avoid this kind of comment?

Reply:
In this case, a separate class is **required** since I need to define a `.erb` template!

Reply:
Why can't we do an inline `call`?

Reply:
the template has html tags, and I don't think we can call them from `call` without calling `html_safe`
I want to avoid calling `html_safe` since it's a smell for me

Also, not sure how `call` would deal with https://github.com/primer/view_components/blob/4ae907d308b955598630430c9982dcdc7b1eafe1/app/components/primer/breadcrumb/item_component.html.erb#L1

Reply:
Not blocking. More for fun, to handle the elements, could we use `content_tag` methods to generate the elements? I'm not sure I understand the white spacing issue.

Reply:
>  I understand the white spacing issue.

Me neither, maybe that's something related with how ERB generates the html when there are `ifs` involved.
I'm going to try some stuff here